connectors/katalog/main.go

```diff
@@ -55,6 +63,18 @@ func RunCmd() *cobra.Command {
 			router.Use(gin.Logger())
 
 			bindAddress := fmt.Sprintf("%s:%d", ip, port)
+			if utils.GetTLSEnabled() {
+				handler.Log.Info().Msg("manager and connector uses TLS")
+				config, err := fybrikTLS.GetServerTLSConfig(&handler.Log, client, utils.GetCertSecretName(), utils.GetCertSecretNamespace(),
+					utils.GetCACERTSecretName(), utils.GetCACERTSecretNamespace(), utils.GetMTLSEnabled())
+				if err != nil {
+					return nil
+				}
+
+				server := http.Server{Addr: bindAddress, Handler: router, TLSConfig: config}
```

Comment:
why do you need this server ?

Reply:
It's how the katalog server is initialized with the tls config and the router.